### ISSUE TITLE
In 8.6.1, ctrl+dragging an existing selection replaces it instead of copying it

### ISSUE THREAD

**Issue by Coises** [notepad-plus-plus/notepad-plus-plus#14561]
### Description of the Issue
When a selection is present, Ctrl+drag beginning within the selection should copy the selection. Instead, it replaces the selection.

### Steps to Reproduce the Issue
1. Open a new tab and type: ```one two three  four```
2. Double-click ```two``` to highlight it.
3. Hold down the Ctrl key, position the mouse cursor over the ```w```, press the left mouse button down, drag to the space between ```three``` and ```four```, release the mouse button then release the Ctrl key.

### Expected Behavior
As in 8.6 and prior versions (and most other editing windows), ```two``` should be copied to the space between ```three``` and ```four```.

### Actual Behavior
The original selection is deselected and the area over which the cursor was dragged is selected.

### Debug Information
Notepad++ v8.6.1   (64-bit)
Build time : Jan  4 2024 - 05:01:52
Path : C:\Program Files\Notepad++\notepad++.exe
Command Line : 
Admin mode : OFF
Local Conf mode : OFF
Cloud Config : OFF
OS Name : Windows 10 Pro (64-bit)
OS Version : 22H2
OS Build : 19045.3803
Current ANSI codepage : 1252
Plugins : none

### Workaround
It is still possible to drag copy by dragging without the Ctrl key (as if you were going to move the text) until the mouse cursor is at the position where you want to place the copy, then pressing the Ctrl key, then releasing the mouse button, then releasing the Ctrl key. However, this is an unexpected change in behavior which differs from the behavior of most edit controls; users may not recognize the need to adopt the new procedure and think it is no longer possible to drag copy at all.

**Comment by mpheath** [comment]
It is a Scintilla change.

See https://sourceforge.net/p/scintilla/bugs/2410/

Committed Ctrl+Click to drop selection with [[3ca229]](https://sourceforge.net/p/scintilla/code/ci/3ca2299633c6ad7595bae095d6547f2f14b94f1d/).

As a consequence, Ctrl+Drag a selection is more differcult now as your workaround suggests.

**Comment by rsakakr** [comment]
Yes, a remarkable regression. Now it's much harder to work with Notepad++.

**Comment by bob454522** [comment]
same issue here :(    I very much liked the prior ctrl + drag method.   please update this if npp is able to fix this (so that i can do the update).    thanks

**Comment by georgiqg** [comment]
I've been having the same issue in v8.6.1 and v8.6.2. I disabled the automatic updates via
Settings => Preferences => MISC. => Enable Notepad++ auto-updater (unchecked)
I also change the "nextUpdateDate" in the config.xml file to a date a few years in the future, so it won't update, but then I save and reopen the file and it is switched back to the current date, so in a few minutes my Notepad++ updates automatically. It's really frustrating.

**Comment by Coises** [comment]
Note that in [8.6.3-RC2](http://download.notepad-plus-plus.org/repository/8.x/8.6.3.RC2/), if the newly-restored setting at **Settings** | **Preferences** | **Editing 2**: **Multi-Editing**: **Enable Multi-Editing (Ctrl+Mouse click/selection)** is *not* checked, the familiar Ctrl+drag behavior is restored.

This seems like a reasonable compromise — and since the source of the new behavior is a change in Scintilla, outside the control of Notepad++, it’s probably the only reasonable compromise possible.

I assume I shouldn’t take it upon myself to close this issue (if I can... I don’t think I’ve ever tried to close my own issue in someone else’s repository) since others have been troubled by the new behavior and commented here as well; however, as for me personally, this is sufficient to close the issue.

**Comment by alankilborn** [comment]
>I assume I shouldn’t take it upon myself to close this issue (if I can...

If you see a Close button, then you can (and should).
You should have a Close button because you are this issue's petitioner.

**Comment by masongbrown** [comment]
is there any word on an 8.6.3 release date?  looking forward to having ctrl-click "restored" to what ive been using for years.

**Comment by alankilborn** [comment]
> is there any word on an 8.6.3 release date (non RC)? looking forward to having ctrl-click "restored" to what ive been using for years.

If you have to ask that then you should just go with the RC.

**Comment by LadyBird13** [comment]
This tip helped me:
setting at Settings | Preferences | Editing 2: Multi-Editing: Enable Multi-Editing (Ctrl+Mouse click/selection) is not checked, 
The familiar Ctrl+drag behavior is restored.

**Comment by Coises** [comment]
This was adequately mitigated in 8.6.3 (as noted in [comment above](https://github.com/notepad-plus-plus/notepad-plus-plus/issues/14561#issuecomment-1948841813)). I should have closed it long ago... doing so now.
